src/helpers/validateSignUpEmailAddress.js

```diff
@@ -0,0 +1,13 @@
+import { ALPHANUMERIC } from "./validateEmailAddress";
+
+const validateSignUpEmailAddress = (email) => {
+    const emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN
```

Comment:
To deal with the possibility the env var isn't set, can you just add `|| ".+"` here, so any email will be allowed locally.

Reply:
Do we have to deal with the env var not being set locally. Shouldn't it be set?

I noticed that when `const emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN` is declared outside the validateSignUpEmailAddress method, `process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN` is `undefined` remotely. Thus making the unit test fail.
Why is this so and adding ".+" would mean that the unit test for validateSignUpEmailAddress will give false positive results?
Well this is my understanding but tell me what you think.

Reply:
As discussed on slack `emailDomain = process.env.NEXT_PUBLIC_SIGN_UP_EMAIL_DOMAIN` is now inside validateSignUpEmailAddress, and adding `|| ".+"`